packages/strapi-design-system/src/Icon/Icon.js

```diff
@@ -4,8 +4,8 @@ import { Box } from '../Box';
 import styled from 'styled-components';
 
 const IconWrapper = styled(Box)`
-  width: ${({ theme, width }) => theme.spaces[width]};
-  height: ${({ theme, height }) => theme.spaces[height]};
+  width: ${({ theme, width }) => theme.spaces[width] ? theme.spaces[width] : width};
+  height: ${({ theme, height }) => theme.spaces[height] ? theme.spaces[height] : height};
```

Comment:
I think you could shorten that:

```suggestion
  width: ${({ theme, width }) => theme.spaces[width] ?? width};
  height: ${({ theme, height }) => theme.spaces[height] ?? height};
```

Reply:
I just saw that `handleResponsiveValues` also allows string values, so what about using it here as well? 
@ronronscelestes @gu-stav

> Should these new props also allow responsive properties just like the padding props in the Box component?

Reply:
mmh good question!
the utility of `handleResponsiveValues` is to allow responsive values depending on media queries we chose - is it something we want to include here 🤔

Reply:
Since the Icon component is based on `Box` and `Box` already handle width and height. I'm wondering if we really need that modification 🤔 

https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/Box/Box.js#L74

Reply:
I agree with you @HichamELBSI maybe the issue is with the documentation we need to display these props in the storybook instead.